### PR TITLE
area-codes by state

### PR DESCRIPTION
**Pull request by juanbandajr17**

### CODE REVIEW

File: spec/lib/areacodes_spec.rb [hunk 1]
```diff
@@ -52,6 +52,14 @@
     end
   end
 
+  describe '.area_codes_by_state_code' do
+    it 'returns area_codes' do
+      codes = described_class.area_codes_by_state_code(1)
+      expected = [205, 251, 256, 334, 938].sort
+      expect(codes.sort).to eql(expected)
```
Comment (evitero) [inline spec/lib/areacodes_spec.rb]: Tiny thing but can use `match_array` (which doesn't care about order) https://www.rubydoc.info/github/rspec/rspec-expectations/RSpec%2FMatchers:match_array to avoid having to sort

File: spec/lib/areacodes_spec.rb [hunk 2]
```diff
@@ -52,6 +52,14 @@
     end
   end
 
+  describe '.area_codes_by_state_code' do
+    it 'returns area_codes' do
+      codes = described_class.area_codes_by_state_code(1)
+      expected = [205, 251, 256, 334, 938].sort
```
Comment (mharkins-spokeo) [inline spec/lib/areacodes_spec.rb]: Maybe consider mocking the data so that the behavior is tested and not the data.  I guess it's unlikely the data will change much, but when it does, it's possible it'll break the test(s).

File: spec/lib/areacodes_spec.rb [hunk 3]
```diff
@@ -52,6 +52,14 @@
...
+      expect(codes.sort).to eql(expected)
+    end
+  end
```
Comment (mharkins-spokeo) [inline spec/lib/areacodes_spec.rb]: Maybe also a test for the negative case, where an empty `Array` is returned?
